tools/spi_nor_config_generator/spi_nor_cfg_gen.py

```diff
@@ -0,0 +1,463 @@
+#!/usr/bin/env python3
+
+# This file is Copyright (c) 2020 Antmicro <www.antmicro.com>
+
+import argparse
+import json
+import os
+import sys
+import re
+from git import Git, Repo
+import progressbar
+import time
+from multiprocessing import Process
+import subprocess
+from linux_spi_nor_extract import linux_extract
+from pull_flashrom_data import dump_flashrom_data
+
+# This file produces JSONs with SPI NOR configs from Flashrom,
+# OpenOCD and Linux SPI NOR driver and generate single JSON file
+# and Python file with classes.
+
+# Some features could be not mentioned in modules we parse from.
+# In case there is a chip which supports a feature but it was not
+# mentioned, we can override config here.
+override_chip_cfg = {
```

Comment:
Can we store this override in a separate file.

Reply:
I moved it right [here](https://github.com/litex-hub/litespi/pull/1/files?file-filters%5B%5D=.patch&file-filters%5B%5D=.py&file-filters%5B%5D=No+extension&file-filters%5B%5D=dotfile&hide-deleted-files=true#diff-0a6347bd6f10ebc4431f61b033d99b5a) and additionally I changed override configs to use enums